src/preact-render-spy.js

```diff
@@ -240,6 +246,31 @@ class FindWrapper {
       .length > 0;
   }
 
+  childAt(index) {
+    verifyFoundNodes(this);
```

Comment:
couldn't this whole method be implemented as `return this.children().at(index)` ?

Reply:
Done